client/src/charts/wrapped_nivo/WrappedNivoCircleProportion/WrappedNivoCircleProportion.js

```diff
@@ -119,101 +38,96 @@ export class WrappedNivoCircleProportion extends React.Component {
     const color_scale = scaleOrdinal().range(newIBCategoryColors);
     const value_formatter = get_formatter(is_money, formatter, true, false);
 
-    const graph_data = {
-      id: parent_name,
-      name: parent_name,
-      // ... nivo bubble will roll child values back up and add them to the parent value for use in the graph
-      // so we need to remove the inner portion from the total here
-      value: parent_value - child_value,
-      color: color_scale(parent_name),
-      children: child_value
-        ? [
-            {
-              id: child_name,
-              name: child_name,
-              value: child_value,
-              color: color_scale(child_name),
-            },
-          ]
-        : 0,
+    const Circles = () => {
+      // arbitrary parent values
+      const minX = 350;
+      const minY = 350;
+      const parent_radius = minX - 330;
+      const parent_cx = 175;
+      const parent_cy = 25;
+
+      // child circle calculations
+      const child_percent = child_value / parent_value;
+
+      // precaution against smaller child_percent values
+      const true_child_radius = parent_radius * child_percent;
+      const default_child_radius = parent_radius * 0.1;
+      const child_radius =
+        true_child_radius > default_child_radius
+          ? true_child_radius
+          : default_child_radius;
+
+      return (
+        <svg
+          viewBox={
+            -parent_cx.toString() +
+            " " +
+            -parent_cy.toString() +
+            " " +
+            minX.toString() +
+            " " +
+            minY.toString()
+          }
+        >
+          <circle
+            r={parent_radius.toString()}
+            fill={color_scale(parent_name)}
+          />
+          <circle
+            cy={(parent_radius - child_radius).toString()}
```

Comment:
See the comment on the inner node y position calculation on the previous version, probably want to keep the same rule of adding a `- 1` offset (and keeping the comment explaining why).

Reply:
Done!